pkg/render/intrusiondetection/dpi/dpi_test.go

```diff
@@ -208,6 +210,12 @@ var _ = Describe("DPI rendering tests", func() {
 		dpiCertSecret, err = certificateManager.GetOrCreateKeyPair(cli, render.DPITLSSecretName, common.OperatorNamespace(), []string{""})
 		Expect(err).NotTo(HaveOccurred())
 
+		snortRules, err := embed.AsConfigMap(
```

Comment:
If you use the applicationlayer embed package, this means that your configmap contents is going to be whatever is in [this](https://github.com/tigera/operator/tree/master/pkg/render/applicationlayer/embed/coreruleset) directory. is this OK?

Reply:
Thanks, changed as per previous comment.